composer.json

```diff
@@ -5,5 +5,8 @@
   "type": "wordpress-plugin",
   "license": "GPL-3.0",
   "minimum-stability": "dev",
-  "require": {}
+  "require": {},
+  "require-dev": {
+    "wp-cli/wp-cli": "^1.3"
```

Comment:
I loaded it as `dev` because WP CLI will be installed separately. It's just helpful for IDEs to have code completion during development, but it's not necessary for production.